.github/workflows/rust.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
```

Comment:
I think that the file name `test.yml` would be better than `rust.yml`. 
Because `rust.yml` do not have explicitly meaning so I can not  predict what this file work.

Reply:
Understood, fixed.